testlib/helper.go

```diff
@@ -40,35 +48,118 @@ func NewMainHelper() *MainHelper {
 
 	utils.TranslationsPreInit()
 
+	return &mainHelper
+}
+
+func NewMainHelper() *MainHelper {
+	mainHelper := newMainHelper()
+	mainHelper.setupStore()
+	return mainHelper
+}
+
+func NewMainHelperWithOptions(options *HelperOptions) *MainHelper {
+	mainHelper := newMainHelper()
```

Comment:
nit: Having three-ish constructor functions seems complex. Could we just have all the logic in `NewMainHelperWithOptions`, and have `NewMainHelper()` invoke `NewMainHelperWithOptions(nil)` instead, eliminating `newMainHelper`?

Reply:
I did this way to make it easy to accommodate changes in NewMainHelperWithOptions without impacting enterprise code, like how it was failing without having the behaviour of NewMainHelper restored to the previous one. No problem changing it the way you suggested.
Does enterprise code use db or store or any other public members of MainHelper?
If yes then I think we need to set `EnableStore` to true.
Please let me know what do you think.

Reply:
I think we can accomplish what you describe above with just the two constructor functions, by writing `NewMainHelper` in terms of the `NewMainHelperWithOptions`:
```go
func NewMainHelper() *MainHelper {
        return NewMainHelperWithOptions(&HelperOptions{
            EnableStore: true,
            EnableResources: true,
        })
}
```

Yes, the enterprise code requires the store, and ideally we enable the resources automatically there as well.